setup/tomcat/conf/conf.d/setmem.conf

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+function calculate_heapsize_mb() {
+    local USAGE_PERCENT METASPACE_SIZE_MB METASPACE_SIZE_KB TOTAL_MEM_KB HEAP_KB HEAP_MB
+    USAGE_PERCENT=$1
+    METASPACE_SIZE_MB=$2
+    TOTAL_MEM_KB="$(awk '/MemTotal:/ { print $2 }' /proc/meminfo)"
+
+    (( METASPACE_SIZE_KB=METASPACE_SIZE_MB / 1024 ))
+    (( HEAP_KB=(TOTAL_MEM_KB - METASPACE_SIZE_KB) * USAGE_PERCENT / 100 ))
+    (( HEAP_MB=HEAP_KB / 1024 ))
+
+    echo "$HEAP_MB"
+}
+
+# Set default and maximum metaspace size
```

Comment:
Add comment:
Recommended settings for 2 OneVizion installations is 
METASPACE_SIZE_MB="384"
METASPACE_MAX_SIZE_MB="512"
Each additional installation require additional 128MB added to the each param

Reply:
Done.

Reply:
Sorry, my typo. Should be:
Recommended settings for 2 OneVizion installations **are**

Reply:
Done.